pkg/tests/ossm/deploy_on_infra_test.go

```diff
@@ -130,7 +130,10 @@ spec:
 			})
 
 			t.LogStep("Verify that the following control plane pods are running on the infra node: istiod, istio-ingressgateway, istio-egressgateway, jaeger, grafana, prometheus")
-			istioPodLabelSelectors := []string{"app=istiod", "app=istio-ingressgateway", "app=istio-egressgateway", "app=grafana", "app=prometheus", "app=jaeger"}
+			istioPodLabelSelectors := []string{"app=istiod", "app=istio-ingressgateway", "app=istio-egressgateway", "app=grafana", "app=prometheus"}
+			if (env.GetSMCPVersion().GreaterThanOrEqual(version.SMCP_2_5)){
```

Comment:
Shouldn't be there `LessThanOrEqual` :thinking:  as it will be removed from SMCP 2.6

Reply:
exactly :man_facepalming: I chose wrong function from tooltip :laughing: , I will fix that